src/apps/testapps/testH3ToCenterChild.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2017-2019 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdlib.h>
+#include "h3Index.h"
+#include "test.h"
+
+SUITE(h3ToCenterChild) {
```

Comment:
It wouldn't hurt to add an exhaustive test here - something like (pseudocode):
```
for res in 0 to 14:
  for childRes in res + 1 to 15:
    h3Index = indexAtRes(res)
    centerChild = h3ToCenterChild(h3Index, childRes)
    // some property-based tests
    t_assert(h3GetResolution(centerChild) == childRes)
    t_assert(h3GetParent(centerChild, res) == h3Index)
```

Reply:
+1, I'd like to see a more exhaustive test for this function

Reply:
Good suggestion. I've also moved here the check that the child should equal indexed centroid of parent at child resolution. Are there more properties that asserted here?

Reply:
Only other check I can think to include is that the returned index passes `h3IsValid`

Reply:
The assertions here look good to me 👍